packages/inertia-vue3/src/link.js

```diff
@@ -0,0 +1,67 @@
+import { h } from 'vue'
+import { Inertia, shouldIntercept } from '@inertiajs/inertia'
+
+export default {
+  functional: true,
```

Comment:
Functional attribute is removed from Vue3
https://v3.vuejs.org/guide/migration/functional-components.html#functional-components

Reply:
Good catch, this has been removed! 👍